src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadataViewImpl.java

```diff
@@ -81,16 +77,26 @@ public EntityMetadataViewImpl(IconsImageBundle iconsImageBundle,
 		this.annotationsWidget = annotationsWidget;
 		this.restrictionWidget = restrictionWidget;
 		initWidget(uiBinder.createAndBindUi(this));
-
 				
 		favoritePanel.addStyleName("inline-block");
 		favoritePanel.setWidget(favoriteWidget.asWidget());
 		
 		doiPanel.addStyleName("inline-block");
 		doiPanel.setWidget(doiWidget.asWidget());
+		annotationsContainer.setWidget(annotationsWidget.asWidget());
+		annotationsContainer.getElement().setAttribute("highlight-box-title", DisplayConstants.ANNOTATIONS);
 	}
 
 	@Override
+	public void setAnnotationsVisible(boolean visible) {
+		if (visible) {
+			annotationsContent.setVisible(true);
```

Comment:
should not need to setVisible(true) (it should already be true, but container collapsed)